custom_components/midea_ac_lan/humidifier.py

```diff
@@ -31,46 +41,38 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 
 
 class MideaHumidifier(MideaEntity, HumidifierEntity):
-    def __init__(self, device, entity_key):
+    def __init__(self, device: Any, entity_key: str) -> None:
         super().__init__(device, entity_key)
 
     @property
-    def target_humidity(self):
-        return self._device.get_attribute("target_humidity")
+    def target_humidity(self) -> float:
+        return cast(float, self._device.get_attribute("target_humidity"))
 
     @property
-    def mode(self):
-        return self._device.get_attribute("mode")
+    def mode(self) -> str:
+        return cast(str, self._device.get_attribute("mode"))
 
     @property
-    def available_modes(self):
-        return self._device.modes
+    def available_modes(self) -> list[str] | None:
+        return cast(list, self._device.modes)
 
-    def set_humidity(self, humidity: int):
+    def set_humidity(self, humidity: int) -> None:
         self._device.set_attribute("target_humidity", humidity)
 
-    def set_mode(self, mode: str):
+    def set_mode(self, mode: str) -> None:
         self._device.set_attribute("mode", mode)
 
     @property
-    def min_humidity(self):
-        return self._min_humidity
+    def is_on(self) -> bool:
+        return cast(bool, self._device.get_attribute(attr="power"))
 
-    @property
-    def max_humidity(self):
```

Comment:
here removed property `min_humidity` and `max_humidity`, make `self._min_humidity` and `self._max_humidity` meaningless.

Reply:
I plan to change `_min_humidity` to `_attr_min_humidity`